docs/conf.py

```diff
@@ -132,6 +132,7 @@
     "sphinxcontrib.apidoc",
     # Tree-local extensions:
     "single_sourced_data",  # in-tree extension
+    "sphinx_copybutton",
```

Comment:
This is a section for in-tree extensions, third-party ones must be listed above, for consistency.